inc/audiences/namespace.php

```diff
@@ -490,31 +490,33 @@ function admin_enqueue_scripts() {
  * @return array|null
  */
 function get_estimate( array $audience ) : ?array {
-	global $wpdb;
 	$since = Utils\date_in_milliseconds( '-1 week', DAY_IN_SECONDS );
 
 	// Append the groups query.
 	$audience_where = build_audience_query( $audience );
```

Comment:
Feels like this should be returning a tuple of query and parameters?
```suggestion
	[ $audience_where, $audience_params ] = build_audience_query( $audience );
```

Then
```php
$query_params = [
	...$audience_params,
	'blog_id' => get_current_blog_id(),
```

Reply:
It's a bit of a PITA with the requirement for building some `ILIKE` queries so swerved it for now. I can revisit though.

Reply:
(Not clear from what the function is actually returning, but this is just a guess)

Reply:
Updated to return a tuple, string and params array.